CHANGES

```diff
@@ -18,6 +18,8 @@ The rules for this file:
   * 1.0.0
 
 Changes
+  - Now AMBER parser raises an exception when an inconsinstency in 
+    the input file is found, instead of ignoring the file (PR #)
```

Comment:
reference issue #238 (and possibly #227); you can also reference PR #256 but if there's an issue, _always_ reference the issue

Reply:
Done!